src/content/1.7/modules/concepts/hooks/list-of-hooks.md

```diff
@@ -1333,7 +1333,7 @@ displayAdminGridTableBefore
 
     This hook adds new blocks before Grid component table.
 
-    Located in: src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/table.html.twig
+    Located in: templates/bundles/PrestaShopBundle/views/Admin/Common/Grid/Blocks/table.html.twig
```

Comment:
Maybe add a note at the begin of this file to tell something like:
"Before the 1.7.9.0, files were inside the `src/PrestaShopBundle/Resources` directory"

Reply:
Now that we have a documentation section for 1.7 and a documentation section for 8, this is not needed anymore I believe 😉